test/serverTest.js

```diff
@@ -7,51 +7,50 @@
 'use strict';
 
 const assert = require('chai').assert;
-const jsonfile = require('jsonfile');
 const async = require('async');
 const config = require('../config.json');
 const request = require('request').defaults({'proxy': config.proxy});
 
-var fold = require('../src/backend/fold.js');
-var generator = require('../src/backend/generator.js');
-var dist = require('../src/backend/dist.js');
-var app = require('../src/app');
+let fold = require('../src/backend/fold_v2.js');
+let generator = require('../src/backend/generator.js');
+let dist = require('../src/backend/dist.js');
+let app = require('../src/app');
+let fs = require('fs');
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['fs' is never reassigned. Use 'const' instead. (prefer-const)](https://app.codacy.com/manual/triplez/open-event-webapp/pullRequest?prid=3226522)